src/adapter/bundle.js

```diff
@@ -42,6 +44,13 @@ define([
                         "$injector"
                     ]
                 }
+            ],
+            policies: [
+                {
+                    category: "composition",
+                    implementation: AdapterCompositionPolicy,
+                    depends: [ "mct" ]
```

Comment:
Yeah, that's in #1107. I think I was counting on that getting merged into #999 first (and didn't replicate changes to avoid merge conflicts)